### PR TITLE
renamed and reformatted md files to rst

### PR DESCRIPTION
**Pull request by reece**
This PR replaces md files with rst, and reformats contents. The goal is to standardize on rst for in-repo docs in addition to the sphinx docs.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+The current code conventions for the source files are as follows:
+
+- Use two-space indentation, and no tabs.
+- Hard-wrap code to 80 characters per line.
+- Use ``UpperCamelCase`` for object or record names.
+- Use ``lowerCamelCase`` for attribute or method names.
+- Use ``CONSTANT_CASE`` for global and constant values.
+- Comments:
+
+- Comments should be indented at the same level as the surrounding
```
Comment (calbach) [inline CONTRIBUTING.rst]: In Markdown, this was a sub-bullet of `Comments:`. In RST this has become a normal bullet:
https://github.com/reece/schemas/blob/documentation/CONTRIBUTING.rst#syntax-style-and-conventions

File: INSTALL.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
+Installing the GA4GH Schemas
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+The schemas are documents (text files) that formally describe the
+messages that pass between GA4GH reference servers and clients, which we
+also refer to collectively as "the API." The schemas are written in a
+language called `Avro <http://avro.apache.org>`__.
+
+We use the schemas in a couple of different ways: - to generate source
```
Comment (calbach) [inline INSTALL.rst]: These were bullets in MD:

https://github.com/reece/schemas/blob/documentation/INSTALL.rst

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+@@@@@@@@@@@@@@@
+
+The `Reads Task
+Team <https://groups.google.com/forum/#!forum/dwgreadtaskteam>`__ is
+focused on standards for accessing genomic read data -- collections of
+primary data collected from sequencing machines.
+
+The team will deliver:
+
+1. Data model. An abstract, mathematically complete and precise model of
```
Comment (calbach) [inline README.rst]: These numbers are no longer on their own line in RST:  https://github.com/reece/schemas/blob/documentation/README.rst

**Comment by calbach** [comment]
I've validated by eyeball that there are no significant formatting or content changes from this PR, aside from the issues noted above.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (reece) [inline CONTRIBUTING.rst]: Fixed. PR coming.

**Comment by reece** [comment]
Thanks for looking @calbach! Sorry I didn't catch those.
All fixed in b804b0b (I hope).

**Comment by diekhans** [comment]
thanks @reece !!

**Pull request merged**